src/analysis/special_functions/stirling.lean

```diff
@@ -203,3 +209,88 @@ begin
   rw ←filter.tendsto_add_at_top_iff_nat 1,
   exact tendsto_at_top_cinfi stirling_seq'_antitone ⟨x, hx'⟩,
 end
+
+/-!
+ ### Part 2
+ https://proofwiki.org/wiki/Stirling%27s_Formula#Part_2
+-/
+
+/-- For `n : ℕ`, define `w n` as `2^(4*n) * n!^4 / ((2*n)!^2 * (2*n + 1))` -/
+noncomputable def w (n : ℕ) : ℝ :=
```

Comment:
You are in the root namespace, which means that you are forbidding the notation `w` to everyone forever with this definition. Instead, I think you should do everything in a `stirling` namespace to avoid that: put `namespace stirling` at the beginning of your file, and `end stirling` at the very end to do this.

Reply:
In
https://github.com/leanprover-community/mathlib/pull/14874#discussion_r948855922
it was suggested to remove the namespace. Putting it back.